iroh-net/src/magicsock/peer_map/endpoint.rs

```diff
@@ -878,7 +872,7 @@ impl Endpoint {
         self.pending_cli_pings.clear();
     }
 
-    fn last_ping(&self, addr: &SendAddr) -> Option<Instant> {
+    pub fn last_ping(&self, addr: &SendAddr) -> Option<Instant> {
```

Comment:
`pub(super)`?

Reply:
the whole `Endpoint` struct is only `pub(super)`, so no need to add it to all methods I thought?

Reply:
changed it all to `pub(super)`